### ISSUE TITLE
Consider upgrading to datauri version 2

### ISSUE THREAD

**Issue by cappslock** [jrit/web-resource-inliner#44]
datauri version 2 contains a fix for a Webpack issue (see here for more context: https://github.com/data-uri/datauri/issues/20)

Upgrading to that version would help folks that are webpacking this package or its dependents to avoid a workaround.

Happy to submit a PR for this if desired. Thanks!

**Comment by jrit** [comment]
yeah, I'd be happy to take a PR to upgrade a dep, especially if it is non-breaking and fixes an issue :)

**Comment by cappslock** [comment]
https://github.com/jrit/web-resource-inliner/pull/45

Thanks for your time.
